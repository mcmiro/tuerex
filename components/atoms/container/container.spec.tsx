import * as React from "react";
import {screen, render} from "@testing-library/react";
import Container from "./";

describe("Test Container component", () => {
	it("Should pass when children is additional", () => {
		render(<Container data-testid="test">test</Container>);

		const renderContainerComponent = screen.queryAllByTestId("test");
		expect(renderContainerComponent).not.toBeNull();
		expect(renderContainerComponent).toBeDefined();
		expect(renderContainerComponent).toMatchSnapshot();
	});
});
