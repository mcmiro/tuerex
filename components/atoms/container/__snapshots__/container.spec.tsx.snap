// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Container component Should pass when children is additional 1`] = `[]`;
